### DISCUSSION TITLE
Can we deploy prisma1 and prisma2 schema to same db of MYSQL.

### DISCUSSION THREAD

**Discussion by rahulybisen** [prisma/prisma#5492] [Q&A]
I am having one application with prisma1  there i am deploying schema to MYSQL.

in another application i am using Prisma2 i want to deploy my schema to same Db which i used to deploy for Prisma1.

I tried same scenario  but at migration time of Prisma2 all table related to prisma1 is deleting and I am getting only prisma2 migration related table.

Help me to handle this scenario.

Thanks

**Comment by janpio** [comment]
This might help here: https://www.prisma.io/docs/concepts/components/preview-features/native-types/native-types-mappings#ignore-and-ignore Note that this currently is a preview feature that explicitly needs to be enabled.

**Reply by rahulybisen** [reply]
Hi @janpio ,

I checked above link shred by you but there I did not got information about to handle schema deployment of prisma1 and prisma2 to same db url. Which I ma having two separate application one with prisma1 and another with prisma2 I want to deploy both schema to some db url without hampering or deleting table's of other application.

**Reply by janpio** [reply]
You can use the feature that I linked to, for Prisma 2 to not mess with the marked tables at all. Alternatively you can also just not use Prisma 2 to manage your database tables at all, and let Prisma 1 do all the migrations and deployment of the database schema.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
